Makefile.am

```diff
@@ -96,3 +96,8 @@ include $(top_srcdir)/makefiles/test-cases.mak
 
 include $(top_srcdir)/makefiles/help.mak
 
+if HAVE_RST2MAN
+%.1.in: %.1.in.rst
```

Comment:
> I expect such person may install both rst2man and GNU make.

That's not really a reason not to make a very simple modification for the rule to work everywhere, is it?